web/docs/language/overview.md

```diff
@@ -73,4 +73,4 @@ psl=}
 
 You can check out a full working example [here](https://github.com/wasp-lang/wasp/tree/main/waspc/examples/todoApp).
 
-In the following sections each of the basic language features is explained. 
+In the following sections, each of the basic language features is explained.
```

Comment:
I don't think this is correct, Grammarly suggested it?

Reply:
![Screenshot 2023-03-02 at 11 51 30](https://user-images.githubusercontent.com/2223680/222408066-0db0a2f9-d9d6-4fe9-9586-123f7546522e.png)

It got confused I guess 🤷